packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -296,6 +324,29 @@ const showChecklist = computed(() => {
   return false
 })
 
+const showFeedbackRequest = computed(() => {
+  let storedDateString = onboardingOrFeedbackDate.value
+
+  const currentDate = new Date()
+
+  if (!storedDateString) {
+    const formattedDate = currentDate.toISOString().split('T')[0]
```

Comment:
Why do this? These manual string manipulations are brittle and you can just store the full timestamp and then easily read it any way you like with dayjs()